### PR TITLE
Fix problem with cards comparison #51

### PR DESCRIPTION
**Pull request by IvanovAndrew**

### CODE REVIEW

**Comment by IvanovAndrew** [comment]
@bashnia007, sometimes Lobby.Get_games_returns_the_actual_list_of_games() test fail at my laptop. It looks like race condition.

File: Domain/Game/CardFactory.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+namespace Domain.Game
+{
+    internal static class CardFactory
+    {
+        private static Random Random = new Random();
+
+        public static BangGameCard Create(Type card)
+        {
+            var suite = (Suite) Random.Next(0, 4);
+            var rank = (Rank) Random.Next(0, 4);
```
Comment (bashnia007) [inline Domain/Game/CardFactory.cs:13]: Do we have only 4 ranks?

Comment (IvanovAndrew) [reply Domain/Game/CardFactory.cs:13]: Oh, it's my fault. I'll fix it

Comment (IvanovAndrew) [reply Domain/Game/CardFactory.cs]: Fix in commit 1686ce

**Pull request merged**